src/FE/src/app/pages/users-page/users-page.component.css

```diff
@@ -0,0 +1,75 @@
+.container {
+  display: flex;
+  flex-direction: column;
+  height: 90vh;
+  width: 80vw;
+  padding-top: 1rem;
+  overflow: auto;
+  transition: width 0.3s ease;
+}
+
+.infoContainer {
+  display: flex;
+  flex-direction: row;
+  max-height: 100vh;
+  max-width: 100%;
+  gap: 1rem;
+  height: fit-content;
+}
+
+.userTitleContainer {
+  display: flex;  /* Align children horizontally */
```

Comment:
kan vi fjerne disse kommentarer... det ligner noget en chatbot ville have lavet. bare lidt for obvious, hvis det rent faktisk er